mlir-clang/Test/Verification/omp2.c

```diff
@@ -0,0 +1,30 @@
+// RUN: mlir-clang %s --function=* -fopenmp -S | FileCheck %s
+
+void square2(double** x, int sstart, int send, int sinc, int tstart, int tend, int tinc) {
+    #pragma omp parallel for collapse(2)
+    for(int i=sstart; i < send; i+= sinc) {
+    for(int j=tstart; j < tend; j+= tinc) {
```

Comment:
fix format?

Reply:
I don't feel strongly that input tests should be formatted, but can do if you think.